controllers/tasks.js

```diff
@@ -134,7 +135,19 @@ const fetchPaginatedTasks = async (query) => {
 
 const fetchTasks = async (req, res) => {
   try {
-    const { status, page, size, prev, next, q: queryString, assignee, title, userFeatureFlag } = req.query;
+    const {
+      status,
+      page,
+      size,
+      prev,
+      next,
+      q: queryString,
+      assignee,
+      title,
+      userFeatureFlag,
+      orphaned,
+      dev,
+    } = req.query;
```

Comment:
Are we using all these query?

Reply:
yeah I only added orphaned and dev, it's prettier change.